lib/src/clojure_lsp/feature/hover.clj

```diff
@@ -88,7 +88,8 @@
                    (if markdown?
                      (docstring->formatted-markdown doc)
                      doc))
-        clojuredocs (f.clojuredocs/find-hover-docs-for sym-name sym-ns db*)]
+        clojuredocs (f.clojuredocs/find-hover-docs-for sym-name sym-ns db*)
+        filename (shared/uri->filename uri)]
```

Comment:
Filename is used for display purposes here.

Reply:
Maybe this one we could show the URI? we would need to check how clients behave with that, I can see relevant if I'm using a remote LSP connection (we could support it in the future), showing the URI would be better than showing a filename I guess

Reply:
We could leave to a next PR too

Reply:
I'll add some TODO comments so we remember to look at this again in a future PR.